rasa/utils/endpoints.py

```diff
@@ -142,12 +148,24 @@ async def request(
             del kwargs["headers"]
 
         url = concat_url(self.url, subpath)
+
+        sslcontext = None
+        if self.cafile:
+            try:
+                sslcontext = ssl.create_default_context(cafile=self.cafile)
```

Comment:
should we also allow `capath`?

Reply:
Any specific reason? Would prefer to keep this change small as we'll be releasing in a micro most likely.

Reply:
Just in case they don't have all certificates in one file but rather in a directory (maybe @mvielkind has some more insights on this?)